gramps/plugins/view/pedigreeview.py

```diff
@@ -197,9 +197,17 @@ def __init__(self, view, format_helper, dbstate, person, alive, maxlines,
             for tag_handle in person.get_tag_list():
                 # For the complete tag, don't modify the default color
                 # which is black (#000000000000)
+                old_black = "#000000000000" #self.__color on new tag
+                new_black = old_black[0:7]
                 tag = dbstate.db.get_tag_from_handle(tag_handle)
-                if tag.get_color() != "#000000000000": # only if the color
-                    self.bgcolor = tag.get_color()     # is not black
+                if tag.get_color() != old_black: # only if the color
```

Comment:
Can't this code be simplified to a single "if" statement?

`if tag.get_color() not in ("#000000", "#000000000000"):`

Reply:
Oh, yes, less narrative but clearer.
Thanks.

Note, maybe hardcoded color values should be limited?
To set [long hexa digit and black color](https://github.com/gramps-project/gramps/blob/master/gramps/gen/lib/tag.py#L62) as default color for new tags 
could be a problem on many places with dark gtk3 themes.

Reply:
Perhaps the default should be `None`, meaning that the tag has not been assigned a colour?